packages/terra-tabs/src/common-tabs/Tab.module.scss

```diff
@@ -127,6 +129,13 @@
     }
   }
 
+  .button:focus {
+    &[data-terra-tabs-show-focus-styles='true'] {
```

Comment:
we can add `box-shadow: none` here to override focus styles of terra-button. which would prevent the blue highlight around dotted outline

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41